src/main/java/Duke.java

```diff
@@ -6,5 +9,36 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke\n");
+        System.out.println(" What can I do for you?\n");
+
+        String array[] = new String[100];
+        int counter = 0; // number of items in the list
+        boolean isExit = false;
```

Comment:
It will be good to replace the variable name for greater readability. It takes time for readers to understand what is the variable for (exit from what?).